adyen/assets/logs/adyen_tests.yaml

```diff
@@ -0,0 +1,261 @@
+id: adyen
+tests:
+  - sample: |-
+      {
+      	"notificationItems": [
```

Comment:
Not sure if it's possible in practice, but if `notificationItems` can have multiple entries, we probably want a sample for that case as well.

Reply:
`notificationItems` contain only one NotificationRequestItem object.